packages/neo-one-server-plugin-network/src/__e2e__/createNetworkTest.test.ts

```diff
@@ -17,4 +20,24 @@ describe('create network', () => {
       expect(description.peers).toBeGreaterThan(5);
     });
   });
+
+  test('metrics received', async () => {
+    await one.until(async () => {
+      const output = await one.execute('describe network test --json');
+
+      const description: Network = one.parseJSON(output);
+      const endpoint = description.nodes[0].telemetryAddress;
+
+      const response = await fetch(endpoint, {
+        method: 'GET',
+      });
+      expect(response.ok).toEqual(true);
+
+      const body = await response.text();
+      expect(body.includes('process_')).toEqual(true);
+      expect(body.includes('nodejs_')).toEqual(true);
+      expect(body.includes('neo_')).toEqual(true);
+      expect(body).toMatchSnapshot();
```

Comment:
This is going to be different every time. Instead you should parse out the metric names and snapshot those.